WebHostLib/tracker.py

```diff
@@ -264,16 +264,17 @@ def get_static_room_data(room: Room):
     multidata = Context.decompress(room.seed.multidata)
     # in > 100 players this can take a bit of time and is the main reason for the cache
     locations: Dict[int, Dict[int, Tuple[int, int, int]]] = multidata['locations']
-    names: Dict[int, Dict[int, str]] = multidata["names"]
```

Comment:
We might want to push that change in 2 parts. Uploading a new game to current archipelago.gg breaks here :(

Reply:
current main should already be broken in regards to webtracker